packages/react-components/react-avatar/src/utils/getInitials.ts

```diff
@@ -34,21 +34,22 @@ const MULTIPLE_WHITESPACES_REGEX: RegExp = /\s+/g;
 // eslint-disable-next-line @fluentui/max-len
 const UNSUPPORTED_TEXT_REGEX: RegExp = /[\u0600-\u06FF\u0750-\u077F\u08A0-\u08FF\u1100-\u11FF\u3130-\u318F\uA960-\uA97F\uAC00-\uD7AF\uD7B0-\uD7FF\u3040-\u309F\u30A0-\u30FF\u3400-\u4DBF\u4E00-\u9FFF\uF900-\uFAFF]|[\uD840-\uD869][\uDC00-\uDED6]/;
 
-function getInitialsLatin(displayName: string, isRtl: boolean): string {
+function getInitialsLatin(displayName: string, isRtl: boolean, firstInitialOnly?: boolean): string {
   let initials = '';
 
   const splits: string[] = displayName.split(' ');
```

Comment:
This is definitely an edge case and I'm not sure how much we want to handle bad input but there are ways in which splitting this way can cause failures when subsequently building up the initials.

```js
// Three spaces between the words
"First   Last".split(" "); // ["First", "", "", "Last"]
```